python/requirements.txt

```diff
@@ -1,8 +1,7 @@
 async-timeout==4.0.2;python_version<"3.11"
-maturin==0.13.0
+maturin==0.14.17 # higher version break the needs structure changes, the name of the project is not the same as the package name, and the naming both glide create a circular dependency - TODO: fix this 
```

Comment:
You use 0.14.14 in CD - do you want to align that?

Please create a GH task for this TODO